packages/awslint/lib/rules/docs.ts

```diff
@@ -71,6 +72,21 @@ docsLinter.add({
   },
 });
 
+docsLinter.add({
+  code: 'no-experimental-apis',
+  message: 'The use of @experimental in not allowed (either directly or via parent class)',
+  eval: e => {
+    if (!isPublic(e.ctx)) { return; }
+    // technically we should ban the use of @experimental in the codebase, but since jsii will marks all properties
+    // of experimental modules as experimental we can't.
+    if (isModuleExperimental(e.ctx.assembly)) {
+      return;
+    }
+    const property = e.ctx.documentable;
```

Comment:
Minor: Not just property right. This also applies to apis, classes, interfaces, etc.

```suggestion
    const sym = e.ctx.documentable;
```